src/constants/notifications.ts

```diff
@@ -1,4 +1,9 @@
+import { window } from "rxjs"
+
 export const AlertText: { [key: string]: string } = {
   permissionGranted: `Go to your settings to disable notifications for 'The Rhapsodies'.`,
   permissionDenied: `You have denied permission for notifications. Want to enable notifications? Go to your settings and allow 'The Rhapsodies' to show notifications.`,
 }
+
+// @ts-ignore
+export const WindowOneSignal = window.OneSignal
```

Comment:
I'm not seeing what uses WindowOneSignal in the codebase, but might be missing the context